packages/recommend-react/src/useFrequentlyBoughtTogether.ts

```diff
@@ -6,25 +6,56 @@ import {
 import { useEffect, useState } from 'react';
 
 import { useAlgoliaAgent } from './useAlgoliaAgent';
+import { useStableValue } from './useStableValue';
 import { useStatus } from './useStatus';
 
-export function useFrequentlyBoughtTogether<TObject>(
-  props: GetFrequentlyBoughtTogetherProps<TObject>
-) {
+export function useFrequentlyBoughtTogether<TObject>({
+  objectIDs: userObjectIDs,
+  queryParameters: userQueryParameters,
+  transformItems: userTransformItems,
+
```

Comment:
I don't think we need these empty lines?

Reply:
Indeed: https://github.com/algolia/recommend/pull/52/commits/ba9a894f368f107f139e53e60255e53e627ba4a2